lib/lightning_web/live/credential_live/oauth_component.ex

```diff
@@ -40,10 +43,13 @@ defmodule LightningWeb.CredentialLive.OauthComponent do
          [
            module: __MODULE__,
            form: @form,
+           action: @action,
+           scopes_changed: @scopes_changed,
```

Comment:
I'm not sure about the reason we pass in `@scopes_changed`, would working that out from the `@form.source` changeset not provide the information needed here?